### ISSUE TITLE
config server firewall menu !! wont show any more 

### ISSUE THREAD

**Issue by medsaif95** [serghey-rodin/vesta#2139]
### Operating System (OS/VERSION):

Ubuntu 18 & Centos 7

### VestaCP Version:

1.0.0.3

### Installed Software (what you got with the installer):

php, apache, nginx, mysql

the Scf mune wont show any more !! 
days goes and our server have issues

**Comment by anton-reutov** [comment]
can you provide some screenshots ?

**Comment by medsaif95** [comment]
![image](https://user-images.githubusercontent.com/15780998/140167864-9f9acc41-b308-4fad-ab2f-07a4ee70c944.png)
The same thing happens in my servers ( Ubuntu & Centos 7)

**Comment by anton-reutov** [comment]
> config server firewall menu !! wont show any more

log updates firewall

**Comment by medsaif95** [comment]
![image](https://user-images.githubusercontent.com/15780998/140169314-5ba3b7bb-49da-404d-9f2e-d6197b97733e.png)
config server firewall  external software as showed in image , it works fine in older versions.

**Comment by ScIT-Raphael** [comment]
csf is a third party software which is probaly not directly related to vesta. Vesta did updated their interface to react, which probaly reset your csf integration. you will probaly need to wait until csf adapted their service to the new interface.

**Comment by medsaif95** [comment]
Tested Now it works with direct link https://X.X.X.X:8083/list/csf/  ,  showed with no vestacp headers 

![image](https://user-images.githubusercontent.com/15780998/140169813-2a0856fb-5f6e-486d-a13f-c6b35f147a3b.png)

**Comment by ScIT-Raphael** [comment]
As expected, probaly you want to point csf to upgrade their integration to fit to the new vesta react interface.

**Comment by medsaif95** [comment]
Updated to the last version 14.11 the problem still exist

**Comment by anton-reutov** [comment]
CSF is not default vesta module

**Comment by ScIT-Raphael** [comment]
As written, because CSF doesnt know about the interface change from vesta - this issue isnt related to it, CSF needs to update their software. Would suggest to close the issue on vesta and open it on csf side.

**Comment by anton-reutov** [comment]
Thank you SciT

**Comment by medsaif95** [comment]
OK . thanks

**Comment by Aetherinox** [comment]
An old issue. However, after the original developer ceased developing CSF and I decided to fork, I have updated the CSF documentation to include VestaCP instructions, as well as information about the current bug with CSF not showing in the header.

I also released a fix in regards to the VestaCP header not showing on the `list/csf` page and buttons not functioning in some browsers. So the VestaCP header is back on the csf page, and the buttons now work.

https://docs.configserver.dev/install/install/#install-vestacp

However, with the current state of VestaCP, I'm not investing a lot of time into this unless activity changes. There are a multitude of open exploits / vulnerabilities with VestaCP, and at this point, I would not recommend running it.

If that changes, great. But not at its present state.
